### PR TITLE
docs: ✏️ Add styling guidelines

### LINKED ISSUE #48: Add scss usage standards to documentation
In there should be things like:
- Configurable styles
- @fontface
- global settings
- slots

### PR DESCRIPTION
**Pull request by SheepFromHeaven**
Fixes #48

### CODE REVIEW

File: docs/styling.md [hunk 1]
```diff
@@ -45,3 +45,53 @@ MyButton.register();
...
+If your styles are dynamic and depending on some values in the javascript context, just use inline styles in your template string:
+
+```js
+import Component from '@biotope/element';
+
+class MyButton extends Component {
+  color: 'orange';
+  render() {
+    return this.html`
+      <style>
```
Comment (timomayer) [inline docs/styling.md]: CSS values set with js is an anti pattern and should only be used as some kind of last resort, do we really want to have that in our docs? If yes i vote for adding a reminder that you should only use inline styles and js generated css styles when ther eis no other possible solution.

Comment (jurekbarth) [reply docs/styling.md]: Can you further explain? Setting style in shadow dom always requires JS. Setting dynamic styles could be either this or using custom css properties which is not supported widely enough, so this is the proposed way :)

Comment (SheepFromHeaven) [reply docs/styling.md]: @timomayer?

Comment (timomayer) [reply docs/styling.md]: @SheepFromHeaven we talked about that in person, can you add the hint we talked about, afterwards we can merge it :)

Comment (jurekbarth) [reply docs/styling.md]: @timomayer can you enlighten us, so we know what needs to be done? 👍

**Pull request merged**